nikola/nikola.py

```diff
@@ -1352,129 +1352,14 @@ def flatten(task):
             'task_dep': task_dep
         }
 
-    def scan_posts(self, really=False, ignore_quit=False, quiet=False):
+    def scan_posts(self):
         """Scan all the posts."""
-        if self._scanned and not really:
+        # FIXME this is temporary while moving things out to a plugin
+        # Why doesn't getPluginByName work????
```

Comment:
Let's add a "rescan_posts" method that clears _scanned and calls scan_posts instead.